Source/System.Management/Pash/ParserIntrinsics/AstBuilder.cs

```diff
@@ -2271,6 +2271,31 @@ CommandParameterAst BuildCommandParameterAst(ParseTreeNode parseTreeNode)
                                            new ScriptExtent(parseTreeNode), requiresArgument);
         }
 
+        IEnumerable<CommandElementAst> PostprocessCommandElements(IEnumerable<CommandElementAst> elements)
+        {
+            var queue = new Queue<CommandElementAst>(elements);
+            while (queue.Count > 0)
```

Comment:
I think there's a couple of places where I used `.Count > 0`. I am using it without LINQ for the performance reasons, but if we prefer LINQ way in Pash code, I will replace them.